### DISCUSSION TITLE
A bundle product created using shopify bundles is not published on a custom app

### DISCUSSION THREAD

**Discussion by vamshiaruru** [Shopify/storefront-api-feedback#227] [Help]
Hi everyone, I have a development store where I installed the bundles app and created a product. On this store, I also created a custom app which has all permissions on Admin and Storefront api scopes. However, I can't publish the bundle product on this custom app at all, that means I am unable to fetch it using storefront api. The product is not draft, it is active. It is published on `Online Store` channel, and the only other cahnnel I can publish it on is `Shopify GraphiQL App` app. 

Any help is appreciated. I am at my wits end as to what I am doing wrong.

Thanks in advance.

**Comment by pedropedruzzi** [comment]
I have the same issue. There is a toggle to enable product bundle publishing [1] on partners console but only for apps that are sales channels. And only public apps can become sales channels. The only alternative are admin-created custom apps, for which publishing bundles is enabled by default. Apparently custom apps form Partners were left out of the bundles party.

[1] https://shopify.dev/docs/apps/build/product-merchandising/bundles/turn-on-publishing
